tests/v2/validation/provisioning/rke1/import_cluster_test.go

```diff
@@ -0,0 +1,296 @@
+	RKEStandaloneConfigFileKey = "rkeStandalone"
+)
+
+type RKEStandaloneConfig struct {
+	Version string `json:"version" yaml:"version"`
+}
+
+type ImportClusterTestSuite struct {
+	suite.Suite
+	session        *session.Session
+	client         *rancher.Client
+	clustersConfig *provisioninginput.Config
+	rkeVersion     string
+}
+
+func (i *ImportClusterTestSuite) TearDownSuite() {
+	i.session.Cleanup()
+}
+
+func (i *ImportClusterTestSuite) SetupSuite() {
+	testSession := session.NewSession()
+	i.session = testSession
+
+	i.clustersConfig = new(provisioninginput.Config)
+	config.LoadConfig(provisioninginput.ConfigurationFileKey, i.clustersConfig)
+	rkeCfg := &RKEStandaloneConfig{}
+	config.LoadConfig(RKEStandaloneConfigFileKey, rkeCfg)
+	i.rkeVersion = rkeCfg.Version
+
+	client, err := rancher.NewClient("", testSession)
+	require.NoError(i.T(), err)
+
+	i.client = client
+
+	i.clustersConfig.NodesAndRolesRKE1 = []nodepools.NodeRoles{provisioninginput.RKE1EtcdPool, provisioninginput.RKE1ControlPlanePool, provisioninginput.RKE1WorkerPool}
+}
+
+func (i *ImportClusterTestSuite) TestImportRKE1Cluster() {
+	require.NotEmpty(i.T(), i.rkeVersion)
+
+	subSession := i.session.NewSession()
+	defer subSession.Cleanup()
+
+	client, err := i.client.WithSession(subSession)
+	require.NoError(i.T(), err)
+
+	for _, k8sVersion := range i.clustersConfig.RKE1KubernetesVersions {
+		for _, nodeProviderName := range i.clustersConfig.NodeProviders {
+			var outb bytes.Buffer
+			var stderrb bytes.Buffer
+
+			clusteName := namegen.AppendRandomString("test-import")
+			externalNodeProvider := provisioning.ExternalNodeProviderSetup(nodeProviderName)
+
+			kubeCfg, err := CreateRKE1StandaloneCluster(client, externalNodeProvider, i.clustersConfig.NodesAndRolesRKE1, k8sVersion, i.rkeVersion)
+			require.NoError(i.T(), err)
+
+			err = ImportRKE1StandaloneCluster(client, clusteName, kubeCfg)
+			require.NoError(i.T(), err)
+
+			logrus.Info("running rke remove...")
+			out := exec.Command("./rke_linux-amd64", "remove", "--force")
+			out.Stdout = &outb
+			out.Stderr = &stderrb
+			err = out.Run()
+			logrus.Infoln(outb.String())
+			logrus.Infoln(stderrb.String())
+			require.NoError(i.T(), err)
+
+			err = os.Remove("rke_linux-amd64")
+			assert.NoError(i.T(), err)
+
+			err = os.Remove("cluster.yml")
+			assert.NoError(i.T(), err)
+		}
+	}
+}
+
+func CreateRKE1StandaloneCluster(client *rancher.Client, externalNodeProvider provisioning.ExternalNodeProvider, nodesAndRoles []nodepools.NodeRoles, k8sVersion string, rkeVersion string) ([]byte, error) {
+	rkeConfig := rke1.RKEConfig
+
+	controlPlane := &nodes.Node{}
+	etcd := &nodes.Node{}
+	worker := &nodes.Node{}
+	var outb bytes.Buffer
+	var stderrb bytes.Buffer
+
+	for _, nodeAndRole := range nodesAndRoles {
+		logrus.Info("creating node...")
+		if nodeAndRole.ControlPlane {
+			cpNodes, err := externalNodeProvider.NodeCreationFunc(client, []string{"--controlplane"}, []int32{int32(nodeAndRole.Quantity)})
```

Comment:
We're deprecating the usage of the EC2 client. Please address @igomez06's comment about AWS SDK. We want to rely on corral and terraform rather than cloud providers' SDKs.